cmd/shipperctl/cmd/backup/backup.go

```diff
@@ -0,0 +1,91 @@
+package backup
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
+)
+
+const (
+	kubeConfigFlagName = "kubeconfig"
+	fileFlagName       = "file"
+)
```

Comment:
It feels like those declaration could be closer to their usage, I cannot find value into declaring them as package constants. Am I missing something?